charts/kubescape-operator/values.yaml

```diff
@@ -164,6 +164,9 @@ global:
     enabled: false
     createEgressRules: false
   overrideRuntimePath: ""
+  overrideDefaultCaCertificates:
+    enabled: false
```

Comment:
let's enable it in snapshot_test.yaml for the `all capabilities` scenario so we can see the effects